eternalcore-core/src/main/java/com/eternalcode/core/feature/warp/WarpConfigRepository.java

```diff
@@ -1,61 +1,126 @@
 package com.eternalcode.core.feature.warp;
 
-import com.eternalcode.commons.bukkit.position.Position;
 import com.eternalcode.commons.bukkit.position.PositionAdapter;
 import com.eternalcode.core.configuration.ConfigurationManager;
 import com.eternalcode.core.configuration.implementation.LocationsConfiguration;
+import com.eternalcode.core.configuration.implementation.WarpsConfiguration;
 import com.eternalcode.core.injector.annotations.Inject;
 import com.eternalcode.core.injector.annotations.component.Service;
+
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
-import panda.std.Option;
 
 @Service
```

Comment:
As Mike told you i would prefer change name of class for example WarpLocationRepository

Reply:
this isn't repo for location but for all data related to warps